### DISCUSSION TITLE
Autohide Expression toggle

### DISCUSSION THREAD

**Discussion by zrksyd** [ScanMountGoat/ssbh_editor#79] [Ideas]
My pull request is a few days old. Is it not going to be merged or should I close it so I can merge the current changes?

**Comment by ScanMountGoat** [comment]
I just merged it. I was busy with some IRL stuff. It's ok to post a new comment or @ me on a PR or issue if you haven't received any feedback in a while. I work on a lot of projects, so I won't always be able to get back to you right away.

**Reply by zrksyd** [reply]
I don't want to bother you about that, but you were committing other stuff already. Also didn't know if you got notifications from comments on pull requests, good to know.
